packages/components/src/components/tabs/tabs.jsx

```diff
@@ -84,24 +84,27 @@ class Tabs extends React.Component {
                     ref={this.setTabsWrapperRef}
                 >
                     {React.Children.map(children, (child, index) => {
-                        const { count, header_content, label } = child.props;
+                        if (child) {
```

Comment:
Try to use early exit pattern. Without extra indentation and braces, code will look a bit cleaner.
```suggestion
                        if (!child) return null;
                        
                        return (...);
```